src/main/java/appeng/parts/misc/AbstractStorageBusPart.java

```diff
@@ -331,10 +330,6 @@ private int createHandlerHash(BlockEntity target) {
         if (target != null) {
             var inv = this.getInventoryWrapper(target);
 
-            if (inv instanceof MEMonitorIInventory h) {
```

Comment:
what about this?

Reply:
Doesn't seem to be useful as interface inventories can be extracted from, so this has no effect in `MEMonitorIInventory`.